Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,8 @@
 function sum(elements) {
+    let numericalValues = elements.filter(item => typeof item === "number");
+    let sumOfValues = numericalValues.reduce((a,b) => a+b, 0);
+    return numericalValues.length > 0 ? sumOfValues : 0;
 }
-
+//const ITEMS = ["a","b"];
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
sum([Infinity, 1]);
sum([Infinity, Infinity]);
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```